react-paw-mailmerge/src/Helpers/CsvFunctions.tsx

```diff
@@ -92,4 +92,62 @@ function NormalizeJsonObjectResult(parsedCsv:ParseResult<object>, options:CsvOpt
    return result;
 }
 
+export function CheckTemplate(templateMessage: string, headers:Array<string>|undefined):Array<string>{
+    let errorMessages = new Array<string>();
+    console.log("Headers: ", headers);
+    var fields = GetFields(templateMessage);
+    for(const i in fields){
+        console.log("FiledName: ", i, fields[i]);
+        if(headers === null || !headers?.includes(fields[i]?.replace("{{", "").replace("}}",""))){
+            errorMessages.push(`Undefined Field - ${fields[i]}`);
+        }
+    }
+
+    //Do we have any {{ or }} values that have not been identified as fields?
+    const dummyObj = {}
+    let modifiedMessage = DoMailMerge(dummyObj, templateMessage);
```

Comment:
Is the purpose of running `DoMailMerge` here to strip away all of the valid fields?